docs/motivation.md

```diff
@@ -1,3 +1,117 @@
+   recovers the affected applications, not just PVs. Compare this to DR
+   solutions that merely protect the data volumes and leave it upto the user to
+   recover the applications.
+
+## DR Background
+
+Recovering from a disaster involves starting the application, any dependent
+services along with the application data on another cluster. The amount of
+downtime a business can tolerate depends on the potential loss of revenue, which
+in turn affects their choice of the disaster recovery solution. Businesses
+understand the tradeoff between the cost of unscheduled downtime vs. the cost of
+the disaster recovery solution. Businesses use a variety of DR solutions
+depending on their needs.
+
+### High Availability (HA) vs Disaster Recovery (DR)
+
+Kubernetes has many HA features built into it:
+
+- Separating the control plane from the worker nodes.
+- Replicating the control plane components on multiple nodes.
+- Load balancing traffic to the cluster's API server.
+- Having enough worker nodes available, or able to quickly become available, as
+  changing workloads warrant it.
+
+HA features address sub-component failure. Whereas, DR is recovering
+applications that were running on a failed cluster.  A cluster may fail when its
+subcomponents fail beyond the HA limits.  A cluster may be considered to be
+failed when it becomes inaccessible to its clients, say due to a network
+failure.
+
+Kubernetes does not have DR built into it.
+
+### Cluster Data vs Volume Data
+
+To protect and recover a Kubernetes cluster, the following have to be protected:
+
+1. The cluster data in etcd: this includes the deployments, services, pods,
+   containers, PVCs, PV, etc., resources.
+1. The volume data contents of PVs.
+
+### DR Clustering Models
+
+#### Stretched Cluster
+
+A single Kubernetes cluster is stretched across data centers. In this model, the
+solution uses the HA features of Kubernetes (node failure tolerance) to deliver
+DR, but an update to the etcd database is synchronously written across the data
+centers, which increases client operation latencies compared to a single
+unstretched Kubenetes cluster that is located within a data center.  Reducing
+the radius of such a stretched cluster will reduce the latency but may not meet
+the needs of physical separation that some businesses require to protect
+themselves from certain disasters like earthquake, flooding, power-failure, etc.
+
+#### Metro DR
+
+One or more peer recovery clusters are located in another data center or
+availability zone within the Metro area or the same region. In this model, each
+of the peer Kubernetes clusters have their own etcd database (different from the
+stretched cluster solution but similar to the Regional DR solution).
+
+#### Regional DR
+
+One or more peer recovery clusters are located in different regions that are
+more than 300 km away. In this model, each of the peer Kubernetes clusters have
```

Comment:
300 km seems very specific, can word this with some latency toleration instead? I.e "when storage synchronous replication latency tolerance is exceeded" or related.